src/hotspot/cpu/ppc/sharedRuntime_ppc.cpp

```diff
@@ -1657,12 +1657,12 @@ static void continuation_enter_cleanup(MacroAssembler* masm) {
   if (CheckJNICalls) {
     // Check if this is a virtual thread continuation
     Label L_skip_vthread_code;
-    __ lwz(R0, in_bytes(ContinuationEntry::flags_offset()), R1_SP);
+    __ ld(R0, in_bytes(ContinuationEntry::flags_offset()), R1_SP);
```

Comment:
Change this back to `lwz` since `_flags` is of type `int`.

Reply:
Doh! Done.